plugins/eclblas/eclblas.hpp

```diff
@@ -48,8 +48,9 @@
 extern "C" {
 #include <cblas.h>
 }
+namespace eclblas {
 
-extern "C" {
+//extern "C" {
   ECLBLAS_PLUGIN_API bool getECLPluginDefinition(ECLPluginDefinitionBlock *pb);
 
```

Comment:
Is the plugin definition access function no longer extern "C"?  It doesn't matter in this case because we don't use it, but if it is supposed to use C linkage then it is a bad example for anyone reading the code.